ksqldb-engine/src/main/java/io/confluent/ksql/physical/pull/HARouting.java

```diff
@@ -118,9 +118,8 @@ public void close() {
     if (anyPartitionsEmpty) {
       LOG.debug("Unable to execute pull query: {}. All nodes are dead or exceed max allowed lag.",
                 statement.getStatementText());
-      throw new MaterializationException(String.format(
-          "Unable to execute pull query \"%s\". All nodes are dead or exceed max allowed lag.",
-          statement.getStatementText()));
+      throw new MaterializationException("Unable to execute pull query. "
```

Comment:
Nice cleanup and consolidation on the outer caller's trace.

Reply:
:) thanks